Envelopes/saull_Peak envelope generator.lua

```diff
@@ -0,0 +1,292 @@
+-- @description Peak envelope generator
+-- @author saul-l
+-- @version 1.01
+-- @changelog Removed console message, which was accidently left for autogen
+-- @about
+--   # Peak envelope generator
+--
+--   Generates envelope based on audio item peaks.
+--   You can think of it as an offline audio source parameter baker.
+--
+--   Requires ReaImGui, but will prompt you to install it and provide ReaPack repo,
+--   if you don't already have it.
+--
+--   Contains built-in documentation in UI.
+--
+--   Only works with FX envelopes at the moment
+
+-- @description Peak envelope generator
+-- @author saul-l
+-- @version 1.01
+-- @about
+--   # Peak envelope generator
+--
+--   Generates envelope based on audio item peaks.
+--   You can think of it as an offline audio source parameter baker.
+--
+--   Requires ReaImGui, but will prompt you to install it and provide ReaPack repo,
+--   if you don't already have it.
+--
+--   Contains built-in documentation in UI.
+--
+--   Only works with FX envelopes at the moment
```

Comment:
(I've removed this duplicate metadata separately in 7b40c23cda1032ec3391b7035b62cc0a7731b279)